core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/worker/promisehandlers/OpenOperatorHandler.scala

```diff
@@ -13,5 +13,6 @@ trait OpenOperatorHandler {
   this: DataProcessorRPCHandlerInitializer =>
   registerHandler { (openOperator: OpenOperator, sender) =>
     dp.operator.open()
+    dp.operatorOpened = true
```

Comment:
this state does not belong to dp. can you move it to operator or operator manager?

Reply:
I plan to remove this for now.